.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "notebooks/v1"]
+	path = notebooks/v1
+	url = https://github.com/Qiskit/textbook.git
+[submodule "notebooks/v2"]
+	path = notebooks/v2
+	url = https://github.com/Qiskit/textbook.git
```

Comment:
just curious should the branch be included in these somewhere?

Reply:
Oops, fixed in [9a6448c](https://github.com/Qiskit/platypus/pull/1845/commits/9a6448ce7894d939f4f35b245c8dbef5714a979c) - thanks